ui/packages/image/src/Sketch.svelte

```diff
@@ -122,7 +158,15 @@
 					ctx.temp.drawImage(value_img, 0, 0);
 					ctx.temp.restore();
 				} else {
-					ctx.temp.drawImage(value_img, 0, 0);
+					const image = get_image_details();
+
+					ctx.temp.drawImage(
+						value_img,
+						image.x,
+						image.y,
+						image.width,
+						image.height
```

Comment:
Looks like there's some code repetition, perhaps refactor out to a separate method?